CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.16.3)
+
+project(submitty_count_ts)
+
+find_package(Boost COMPONENTS system filesystem REQUIRED)
+
+add_executable(${PROJECT_NAME} src/main.cpp vendor/tree-sitter-python/src/parser.c vendor/tree-sitter-c/src/parser.c src/counter.cpp src/parser.cpp src/utils.cpp vendor/tree-sitter-python/src/scanner.cc)
```

Comment:
I'm not the best with CMake, but I believe that these vendor files could be compiled into an external library like Boost.